schema/schema.sql

```diff
@@ -43,3 +43,7 @@ CREATE TABLE IF NOT EXISTS User (
 );
 
 CREATE INDEX user_primary_calendar_idx ON User (primary_calendar);
+
+ALTER TABLE Meeting DROP FOREIGN KEY organizing_event_fk;
+ALTER TABLE Meeting DROP INDEX meeting_organizing_event_idx;
+ALTER TABLE Meeting ADD COLUMN room_name VARCHAR(50) NULL;
```

Comment:
room's still have an id as PK, why not use the PK as the reference from the meeting record?